foundation-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -686,37 +536,36 @@ Identity.getExperienceCloudId(new AdobeCallback<String>() {
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x)" %}
 ### getExperienceCloudId
 
-{% hint style="info" %}
-Method `getExperienceCloudIdWithCompletionHandler` was added in ACPCore version 2.5.0 and ACPIdentity version 2.2.0.
-{% endhint %}
-
-This API retrieves the ECID that was generated when the app was initially launched and is stored in the ECID Service.
-
-This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall. The values are returned via the callback.
-
 **Syntax**
 
-```objectivec
-+ (void) getExperienceCloudId: (nonnull void (^) (NSString* __nullable experienceCloudId)) callback;
-+ (void) getExperienceCloudIdWithCompletionHandler: (nonnull void (^) (NSString* __nullable experienceCloudId, NSError* __nullable error)) completionHandler;
+```swift
+@objc(getExperienceCloudId:)
+static func getExperienceCloudId(completion: @escaping (String?, Error?) -> Void)
 ```
 
-* _callback_ is invoked after the ECID is available.
-* _completionHandler_ is invoked with _experienceCloudId_ after the ECID is available, or _error_ if an unexpected error occurs or the request times out. The returned `NSError` contains the [ACPError](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/mobile-core-api-reference#acperror) code of the specific error. The default timeout is 500ms.
+* _completion_ is invoked with _String_ after the ECID is available, or _Error_ if an unexpected error occurs or the request times out. The returned `Error` contains the [AEPError](../mobile-core-api-reference#aeperror) code of the specific error. The default timeout is 1000ms.
 
 **Examples**
 
+**Swift**
+
+```swift
+Identity.getExperienceCloudId { ecid, error in
+  if let error = error {
```

Comment:
should also check for error != AEPError.none

Reply:
Same comment as before, the API doesn't return `AEPError.none`.